programs/sll_remove_dups.py

```diff
@@ -0,0 +1,48 @@
+class Node:
```

Comment:
No need to implement `Node` class again. Import `Node` from `single_linked_lists` instead.